Frameworks/CoreText/DWriteWrapper.mm

```diff
@@ -403,15 +436,16 @@ static void __GetGlyphRunDetails(_CTTypesetter* ts, CFRange range, CGRect frameS
 
             // TODO::
             // This is a temp workaround until we can have actual glyph origins
-            run->_glyphOrigins.emplace_back(CGPoint{ xPos, yPos });
-            for (int index = 0; index < glyphRunDetails._dwriteGlyphRun[i].glyphCount - 1; index++) {
-                run->_glyphOrigins.emplace_back(CGPoint{ xPos + glyphRunDetails._dwriteGlyphRun[i].glyphAdvances[index], yPos });
+            for (int index = 0; index < glyphRunDetails._dwriteGlyphRun[i].glyphCount; index++) {
+                run->_glyphOrigins.emplace_back(CGPoint{ xPos, yPos });
+                run->_glyphAdvances.emplace_back(CGSize{ glyphRunDetails._dwriteGlyphRun[i].glyphAdvances[index], 0.0f });
```

Comment:
>0.0f [](start = 114, length = 4)

This should be yPos right? Or does the reference platform has it all offset to 0.0?